### ISSUE TITLE
Default prop value for enum includes quotes in value

### ISSUE THREAD

**Issue by bsmith-cycorp** [reactjs/react-docgen#149]
Example:
```
"props": {
    ...
    "side": {
        "type":{
            "name": "string"
        },
        "required": false,
        "description":"Which side of the parent element the tooltip should appear on.",
        "defaultValue":{
            "value": "'bottom'",
            "computed":false
        }
    }
    ...
}
```

As you can see, `defaultValue.value` has the value `'bottom'` instead of `bottom`, given the following:
```
  getDefaultProps: function() {
    return {
      side: 'bottom'
    };
  },
```

**Comment by danez** [comment]
This is expected right now, afaik. value always contains the raw value, if it is a string also with quotes, bool will be a string containing `true`, number a string with the number `1`,...


Also if you look at the [Readme](https://github.com/reactjs/react-docgen#result-data-structure) the structure says that defaultValue->value is always a string.

**Comment by bsmith-cycorp** [comment]
I don't understand what the reasoning is for that, but more importantly I don't understand how we're supposed to cleanly convert from the "stringified" value to a usable one.

**Comment by fkling** [comment]
> I don't understand what the reasoning is for that

react-docgen's purpose is to extract information for documentation purposes. To make it easier to show a representation of the default value, `value` contains the raw representation. Besides that, react-docgen generates JSON and not every JavaScript value can be encoded as JSON (e.g. functions).

> I don't understand how we're supposed to cleanly convert from the "stringified" value to a usable one

What are you trying to do with that?

**Comment by bsmith-cycorp** [comment]
I guess the function example is a valid use case.

I'm using the json output as a React prop passed into a documentation app, which uses it to render all the documentation in a nice view. I ran into trouble when trying to set up demo instances of each documented component, preconfigured with their default properties. I ended up having to do this:
```
var defaultValue = prop.defaultValue.value;
if(prop.type.name === 'string') {
    defaultValue = defaultValue.substr(1, defaultValue.length - 2);
}
```
which works, but seems pretty fragile
